attachment_logging/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Module automatically log in chatter  the date and time when each attachment was added and which user linked or unlinked files to attachments.
```

Comment:
Create date and user are displayed next to attachments in chatter.
When attachment is added or removed a note is logged in chatter if the corresponding  option is enabled in the General Settings.